modules/ismp/pallets/token-gateway/src/benchmarking.rs

```diff
@@ -0,0 +1,191 @@
+    where
+        T: Config,
+        <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::AssetId: From<H256>,
+        <<T as Config>::NativeCurrency as Currency<T::AccountId>>::Balance: From<u128>,
+{
+    TeleportParams {
+        asset_id: <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::AssetId::from(
+            H256::zero(),
+        ),
+        destination: StateMachine::Evm(100),
+        recepient: H256::from([1u8; 32]),
+        amount: 1100000000u128.into(),
+        timeout: 10,
+        token_gateway: vec![1, 2, 3, 4, 5],
+        relayer_fee: 1000000002u128.into(),
+    }
+}
+
+fn create_dummy_asset<T: Config>(
+    asset_details: GatewayAssetRegistration,
+) -> AssetRegistration<AssetId<T>>
+    where
+        <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::AssetId: From<H256>,
+{
+    AssetRegistration { local_id: H256::zero().into(), reg: asset_details }
+}
+
+#[benchmarks(
+    where
+    T: pallet_balances::Config<Balance = u128>,
+    T: pallet_assets::Config<AssetIdParameter = sp_core::H256,Balance = u128>,
+    <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::AssetId: From<H256>,
+    <<T as Config>::NativeCurrency as Currency<T::AccountId>>::Balance: From<u128>,
+    <T as frame_system::Config>::AccountId: From<[u8; 32]>,
+    u128: From<<<T as Config>::NativeCurrency as Currency<T::AccountId>>::Balance>,
+    <T as pallet_ismp::Config>::Balance: From<<<T as Config>::NativeCurrency as Currency<T::AccountId>>::Balance>,
+    <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::Balance: From<<<T as Config>::NativeCurrency as Currency<T::AccountId>>::Balance>,
+    <<T as Config>::Assets as fungibles::Inspect<T::AccountId>>::Balance: From<u128>,
+    [u8; 32]: From<<T as frame_system::Config>::AccountId>,
+    <T as frame_system::Config>::RuntimeOrigin: From<frame_system::RawOrigin<AccountId32>>,
+)]
+mod benches {
+    use super::*;
+
+    #[benchmark]
+    fn create_erc6160_asset() -> Result<(), BenchmarkError> {
+        let account: T::AccountId = whitelisted_caller();
+
+        let asset_details = GatewayAssetRegistration {
+            name: BoundedVec::try_from(b"Spectre".to_vec()).unwrap(),
+            symbol: BoundedVec::try_from(b"SPC".to_vec()).unwrap(),
+            chains: vec![StateMachine::Evm(100)],
+            minimum_balance: Some(10),
+        };
+        let asset = create_dummy_asset::<T>(asset_details);
+
+        // Set balances
+        let ed = <T as pallet_balances::Config>::ExistentialDeposit::get();
+
+        // Adjust total issuance
+        pallet_balances::Pallet::<T>::force_adjust_total_issuance(
+            RawOrigin::Root.into(),
+            AdjustmentDirection::Increase,
+            ed * 1000,
+        )?;
+
+        let acc = <<T as frame_system::Config>::Lookup as StaticLookup>::unlookup(account.clone());
+
+        pallet_balances::Pallet::<T>::force_set_balance(RawOrigin::Root.into(), acc, ed * 100u128)?;
```

Comment:
Instead of pulling in the `pallet-balances` crate, just use `<<T as pallet_ismp::Config>::Currency as frame_support::fungible::Mutate>::set_balance` to set the account balance.